test/jasmine/tests/mapbox_test.js

```diff
@@ -983,7 +983,9 @@ describe('@noCI, mapbox plots', function() {
 });
 
 describe('@noCI, mapbox toImage', function() {
-    var MINIMUM_LENGTH = 1e5;
+    // decreased from 1e5 - perhaps chrome got better at encoding these
+    // because I get 99330 and the image still looks correct
+    var MINIMUM_LENGTH = 8e4;
```

Comment:
Great. As long as the image looks ok! Some background info -> https://github.com/plotly/plotly.js/pull/1598